tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -373,6 +362,31 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     return OkStatus();
   }
 
+  Status ExchangeBinaryOpAndSlice(HloInstruction *instr, HloInstruction *slice,
```

Comment:
It's not clear to me what `instr` is.  I guess it's the binary op?  Maybe call it `binary_op` instead?